CsharpStudy.2/TIL/DataSource.md

```
@@ -0,0 +1,138 @@
+    		}
+    }
+    ```
+
+5. 데이터 소스의 기능 정의
+    - 데이터 소스에 대한 추상화를 담당하는 인터페이스 정의
+    - 기본적인 기능을 제공하지만, 실제 데이터가 어디에 저장되는지 신경쓰지 않음
+
+    ```csharp
+    public interface IDataSource
+    {
+    		public Task<List<Person>> getPeopleAsync();
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 5

(MD010, no-hard-tabs)